charts/spire/charts/tornjak-ui/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for Tornjak UI (Frontend).
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  # registry: gcr.io
+  # repository: spiffe-io/tornjak-fe
+  registry: ghcr.io
+  repository: spiffe/tornjak-fe
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion.
+  version: "v1.0.1"
+
+imagePullSecrets: []
+nameOverride: ""
+namespaceOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+#labels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 3000
+
+# tornjak - Tornjak specific configuration
+tornjakFrontend:
+  # -- URL of the Tornjak APIs (backend).
+  # The default value is used when both deployed in the same, local cluster.
+  apiServerURL: "http://localhost:10000/"  # 👈 Use it for minikube or kind
```

Comment:
This assumes the Backend runs in the same cluster as Frontend. What if the Backend runs in a different or remote cluster? We can always use Ingress and some other communication means, but I would handle this by some future PRs. I think for what we need today, the way it is done here is sufficient. Users can also test it by accessing the APIs directly from outside of the minikube, assuming the port-forwarding is enabled.

Reply:
This is a tricky one, yeah....

localhost is kind of ugly, but works with port forwarding. Which is the only thing that works right now.

"http://spire-server.spire-system.cluster.local:10000/" doesn't work unless your browser is within the kubernetes cluster, as the browser is the one talking to the apiServer not the frontend pod.

Ingress is probably the more correct answer long term but that has been controversial so should probably come in its own pr.

Almost thinking it might be better to make it a mandatory setting? Ie, don't set it to anything, and in the template put a required section in to block the install if the user didnt set it? https://austindewey.com/2018/12/28/helm-tricks-input-validation-with-required-and-fail/

Reply:
I think the chart should have defaults for running in the same cluster.

What we put in values.yaml should be a sane value for a default install of the chart. If it runs somewhere else people can override the default value.

Reply:
+1 to sane defaults if we can figure some out. But, it cant run in cluster. The users browser is outside the cluster. The website hands static code over to the browser and then the browser is making ajax calls to the backend directly.

Reply:
aaaah ✅ thanks for clarifying. In that case maybe align with the other dns things as we specified `example.org` in other places.